src/rust/ensogl/lib/core/src/gui/component.rs

```diff
@@ -58,139 +64,163 @@ impl MouseTarget for ShapeViewEvents {
     fn mouse_out  (&self) -> &frp::Source { &self.mouse_out  }
 }
 
+impl Default for ShapeViewEvents {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
In ther places the `new` function calls `default`. Let's think some day if we want to unify this.

Reply:
I'm sorry, I did not understand this comment, what do you mean?